### DISCUSSION TITLE
How to consolidate topicref into one html page under the topichead in HTML5 output

### DISCUSSION THREAD

**Discussion by schep** [dita-ot/dita-ot#4437] [Q&A] [answered]
How to generate HTML5 with all topicref under topichead as one page instead of one page per topicref.  Using DITA to generate a book of Family Stories. The topichead is the family member and the topicref is specific stories about this member.  PDF output format is excellent but HTML5 creates too many links. Only need one per family member.

This is example of stories for family member.  Prefer one link for Frederick with all stories done in line similar to PDF output.

![image](https://github.com/dita-ot/dita-ot/assets/6537812/1a638714-0713-4e38-987c-96bc9dbeb786)

**Comment by stefan-jung** [comment, marked as answer]
@schep, I think you would like to use [chunking](https://www.oxygenxml.com/dita/1.3/specs/archSpec/base/chunking.html).

**Reply by schep** [reply]
Thank you Stefan. I'm obviously new with DITA and your suggestion is accomplishing the desired effect.
